backend/api/routers/capture_ws.py

```
@@ -289,16 +292,20 @@ def _chunks_to_wav(chunks: list[bytes]) -> str | None:
-    except Exception as e:
-        logger.warning("ffmpeg conversion failed: %s", e)
+        # ffmpeg succeeded — clean up input, return WAV
         try:
-            os.unlink(tmp_out.name)
+            os.unlink(tmp_in.name)
         except OSError:
             pass
-        return None
-    finally:
+        return tmp_out.name
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)